demo/extra.cc

```diff
@@ -163,6 +163,38 @@ Box<WalletconnectClient> make_new_client(std::string filename) {
     }
 }
 
+Box<Walletconnect2Client> make_new_client2(std::string filename) {
```

Comment:
Maybe we can add some documentation here as well as for other functions? e.g. 
/**
 * @summary 
 *
 * @param
 * @return 
 * @throws 
 */

Reply:
ok